docs/tutorials/intro_tutorial.ipynb

```diff
@@ -1230,8 +1254,6 @@
    },
    "outputs": [],
    "source": [
-    "import pandas as pd\n",
-    "\n",
     "results_df = pd.DataFrame(results)\n",
```

Comment:
This requires pandas. Are you sure it's fine to remove the import above?

Reply:
Yes, Pandas is imported in the beginning so this is redundant and not needed.